src/app.js

```diff
@@ -106,10 +127,15 @@ app.get(
     scope: [
       //Facebook scopes required to publish posts and list pages
       'pages_show_list',
+      'read_insights',
       'pages_read_engagement',
       'pages_manage_posts',
+      'pages_read_user_content',
```

Comment:
What is this one for? For reading comments?

Reply:
Oh, I see it's a requirement: https://developers.facebook.com/docs/graph-api/reference/v12.0/page/feed#read